.circleci/config.yml

```diff
@@ -31,9 +32,39 @@ jobs:
         DD_BIND_HOST: "0.0.0.0"
         DD_API_KEY: invalid_key_but_this_is_fine
     - image: memcached:1.5.9
+    - image: confluentinc/cp-zookeeper:5.0.0
+      environment:
+        ZOOKEEPER_CLIENT_PORT: "2181"
+    - image: confluentinc/cp-kafka:5.0.0
+      environment:
+        KAFKA_ZOOKEEPER_CONNECT: localhost:2181
+        KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9092
+        KAFKA_LISTENERS: PLAINTEXT://0.0.0.0:9092
+        KAFKA_CREATE_TOPICS: gotest:1:1
+        KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: "1"
 
     steps:
     - checkout
+
+    - restore_cache:
+        keys:
+        - v1-librdkafka-v0.11.5
+    - run:
+        name: Install rdkafka
```

Comment:
What is `rdkafka` for? Does the Docker image not contain this part?

Reply:
The confluent kafka client is a cgo wrapper around the rdkafka library, so it has to be installed before it can be used.

Reply:
Ok, yucky